packages/ember-testing/tests/helpers_test.js

```diff
@@ -1074,7 +1074,7 @@ if (!jQueryDisabled) {
         customEvent('ajaxComplete', xhr);
         assert.equal(
           pendingRequests(),
-          1,
```

Comment:
If you see all the customEvents and include the reset for `setupForTesting` we should end with 0 pendingRequests **not** 1.

Reply:
I am not sure why this test passed on a few of the travis scenarios.  I pushed this fix, we'll see where travis lands.